packages/brick-kit/src/index.ts

```diff
@@ -32,3 +32,4 @@ export * from "./checkIf";
 export { useCurrentTheme, useCurrentMode } from "./themeAndMode";
 export * from "./featureFlags";
 export * from "./core/StoryboardFunctionRegistryFactory";
+export * from "./core/MockRegistry";
```

Comment:
我看只有 `getMockRule` 是需要对外的，不需要的不要暴露。

```suggestion
export { getMockRule } from "./core/MockRegistry";
```